src/mmw/mmw/settings/layer_settings.py

```diff
@@ -197,6 +197,8 @@
                         'HUC-8 to analyze.',
             'minZoom': 7,
             'selectable': True,
+            'searchable': True,
+            'search_priority': 100,
```

Comment:
Where did these priorities come from? Would it do just as well to sort these alphabetically on `code`?